solution/deps.yaml

```diff
@@ -16,7 +16,7 @@ cloudserver:
   sourceRegistry: ghcr.io/scality
   dashboard: cloudserver/cloudserver-dashboards
   image: cloudserver
-  tag: 9.0.7-hotfix.1
+  tag: 9.0.7-hotfix.2
```

Comment:
missing the update to the `VERSION` file...

Reply:
done